SU-SWS/stanford_profile
# D8CORE-2196: Button stacking fix for horizontal alignment.

# READY FOR REVIEW

# Summary
The previous approach to stacking buttons didn't take into account horizontal alignment (left, right, center). To allow easy horizontal alignment, this switches things up.
1) by default, all buttons within a <p></p> will display in-line
2) If you want your buttons to be closely stack, you'll need to add the class "su-stack-button"
3) this allow you to have your buttons horizontally aligned left, right, and center both stacked and in-line.

![button-alignment-cardinalsites lndo site-2020 06](https://user-images.githubusercontent.com/284440/83594655-9f5fb500-a514-11ea-8356-76c428141a4c.png)

# Needed By (Date)
- for next release

# Urgency
- Fixes broken stuff

# Steps to Test

1. Check out this branch
2. Clear cache
3. Build examples with buttons:

- left, right, center
- stacked
- inline

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2196
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
